### PR TITLE
Add question 52

### PR DESCRIPTION
**Pull request by asbillt**

### CODE REVIEW

File: front-end-development/front-end-development-quiz.md [hunk 1]
```diff
@@ -495,3 +495,12 @@ console.log(currencies);
 - [ ] This will stop the CCS being loaded on slow data connections.
 
 [Source: StackOverflow](https://stackoverflow.com/a/46750893)
+
+#### Q52. The flex property is often applied to flex items with a value of 1. Using flex: 1 is a shorthand - what does it unpack to?
+
+- [ ] flex: 0 0 100;
+- [ ] flex: 1 0 0;
+- [ ] flex: 0 0 auto;
+- [x] flex: 1 1 auto;
```
Comment (vanekbr) [inline front-end-development/front-end-development-quiz.md:504]: This one is incorrect, `flex:1` is equivalent to `flex: 1 1 0`. Check the link you provided.
`flex: 1 1 auto` comes from `flex: auto` : https://css-tricks.com/almanac/properties/f/flex/#flex-auto

Did you take a screenshot of the original question? It seems that options have no correct answer to the provided question.

**Review by vanekbr — CHANGES_REQUESTED** [review]
Please check the comment

File: front-end-development/front-end-development-quiz.md (hunk 1 above)
Comment (asbillt) [reply front-end-development/front-end-development-quiz.md:504]: I have a screenshot from the Linkedin skill quiz.

Capture3

**Comment by asbillt** [comment]
> Please check the comment

I uploaded the screenshot. I submitted this to Linkedin support. When I get an update from them I will include it here. Thank you for the help.

**Comment by asbillt** [comment]
I went over to [CSS documentation: flex](https://devdocs.io/css/flex) and Flex: 1 and Flex: 1 1 0 and Flex: 1 0 0 are giving me equivalent results. See images
Capture
Capture2
:
Capture3

I am updating the answer to reflect Flex: 1 0 0.

**Comment by asbillt** [comment]
@vanekbr Answer is updated to reflect Flex: 1 0 0. Please let me know if there is anything additional I need to do.

**Review by Ebazhanov — APPROVED** [review]
Looks good to me 👍 , thank you @asbillt!

**Comment by Ebazhanov** [comment]
@all-contributors please add @asbillt for design, code

**Pull request merged**